frontend/src/scenes/project/Settings/TestAccountFiltersConfig.tsx

```diff
@@ -14,13 +16,14 @@ export function TestAccountFiltersConfig(): JSX.Element {
                 <PropertyFilters
                     pageKey="testaccountfilters"
                     propertyFilters={user?.team?.test_account_filters}
-                    onChange={(filters: FilterType[]) =>
+                    onChange={(filters: FilterType[]) => {
                         userUpdateRequest({
                             team: {
                                 test_account_filters: filters,
                             },
                         })
-                    }
+                        reportTestAccountFiltersUpdated(filters)
+                    }}
```

Comment:
nit and somewhat unrelated to the PR - I guess we could now extract this `onChange` to a separate func to limit how crowded component props get